### ISSUE TITLE
List of instances disappeared

### ISSUE THREAD

**Issue by Alterak** [Chocobozzz/PeerTube#3528]
Unfortunately, no one answered me on the [forum](https://framacolibri.org/t/list-of-instances-disappeared/10423) for a very long time. Sorry for duplicating the question.

After upgrading to version 3.0.0 RC1, all the instances that I follow and that are following me have disappeared. Although this function is enabled in the settings. Is this a bug, or was it done on purpose in the pre-release?

![123](https://user-images.githubusercontent.com/11453567/103331942-2bb96280-4a79-11eb-836f-28de98ff74cf.png)

**Comment by Chocobozzz** [comment]
Hello,

6 days especially in Christmas is not a *very long time* ;)

Please provide web browser logs

**Comment by Alterak** [comment]
> Please provide web browser logs

```
Unchecked runtime.lastError: The message port closed before a response was received.
main-es2015.62e8fa9fa3cd638369e4.js:3 Injecting 1 themes.
main-es2015.62e8fa9fa3cd638369e4.js:3 Loading script /plugins/custom-links/0.0.10/client-scripts/dist/common-client-plugin.js of plugin custom-links.
main-es2015.62e8fa9fa3cd638369e4.js:3 Loading script /plugins/social-sharing/0.2.0/client-scripts/dist/common-client-plugin.js of plugin social-sharing.
main-es2015.62e8fa9fa3cd638369e4.js:3 Client credentials loaded.
main-es2015.62e8fa9fa3cd638369e4.js:3 Enabling default theme.
main-es2015.62e8fa9fa3cd638369e4.js:3 Running hook action:router.navigation-end of plugin custom-links.
main-es2015.62e8fa9fa3cd638369e4.js:3 Running hook action:router.navigation-end of plugin social-sharing.
23-es2015.45c4f73494882f28a937.js:1 WebSocket connection to 'wss://peervideo.ru/socket.io/?accessToken=3d073eb243141d26f3ece426d644c662210138f3&EIO=4&transport=websocket&sid=acoOWbMcq3PeoTgbAAAa' failed: Error during WebSocket handshake: Unexpected response code: 400
```

**Comment by Findus23** [comment]
I'm pretty sure this is a general bug in the release candidate (that's what they are for after all) and has already been fixed by https://github.com/Chocobozzz/PeerTube/commit/c5ac9fe7c164741b2497282eac9b968f222fa151 (I haven't tested it, but it sounds very similar to the issue I see).

**Comment by Alterak** [comment]
> I'm pretty sure this is a general bug in the release candidate (that's what they are for after all) and has already been fixed by [c5ac9fe](https://github.com/Chocobozzz/PeerTube/commit/c5ac9fe7c164741b2497282eac9b968f222fa151) (I haven't tested it, but it sounds very similar to the issue I see).

It probably is. I saw this error on other instances that were updated to version 3.0.0 RC1

**Comment by Chocobozzz** [comment]
Yep this stable release should fix this issue. Thanks @Findus23
